pkg/Microsoft.ML.Parquet/Microsoft.ML.Parquet.nupkgproj

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <NoBuild>true</NoBuild>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PackageDescription>A set of Apache Parquet components for ML.NET.</PackageDescription>
```

Comment:
Looking for better wording, if anyone has some feedback.

Reply:
I suppose if I had to try to improve it, I'd point out that these aren't "Apache Parquet components" (I don't know that there's any such animal as that) but rather "ML.NET components". So maybe: "ML.NET components for Apache Parquet support." It's a bit shorter in addition to being, I suspect, slightly more precise.

---
In reply to: [187427255](https://github.com/dotnet/machinelearning/pull/115#discussion_r187427255) [](ancestors = 187427255)

Reply:
Updated.

Reply:
It's not big deal though IMHO

---
In reply to: [187460198](https://github.com/dotnet/machinelearning/pull/115#discussion_r187460198) [](ancestors = 187460198,187427255)